pyscriptjs/Makefile

```diff
@@ -10,8 +10,25 @@ CONDA_EXE := conda
 CONDA_ENV ?= ./env
 env := $(CONDA_ENV)
 conda_run := $(CONDA_EXE) run -p $(env)
+GOOD_NODE_VER := 14
+GOOD_NPM_VER := 6
+NODE_VER    := $(shell node -v | cut -d. -f1 | sed 's/^v\(.*\)/\1/')
+NPM_VER     := $(shell npm -v | cut -d. -f1)
+
+GOOD_NODE   := $(shell if [ $(NODE_VER) -ge $(GOOD_NODE_VER) ]; then echo true; else echo false; fi)
+GOOD_NPM    := $(shell if [ $(NPM_VER) -ge $(GOOD_NPM_VER) ]; then echo true; else echo false; fi)
+
+.PHONY: check-node
```

Comment:
This is a question because I don't know: why don't we use `.PHONY` in all commands?

Reply:
Here's a bit more on what a `.PHONY` target is. Honestly, I've never really been super strict about it... that's probably why 😄  . We could actually (most of the cmds at least), but I wouldn't do it in this PR. Are you ok to merge as is?

Reply:
Let's do it :)